File: C++/705A. Hulk.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n, i;
    string s;
    cin>>n;
    for(i=1; i<=n; i++){
        if(i&1){
            cout<<"I hate ";
        }
        else{
            cout<<"I love ";
        }
        if(i+1<=n){
            cout<<"that ";
        }
    }
    cout<<"it"<<endl;
    return 0;
}

File: Python/705A. Hulk.py
def main():
    n = int(input())
    for i in range(1, n+1):
        if i&1:
            print(end = "I hate ")
        else:
           print(end = "I love ")
        if i+1<=n:
           print(end = "that ")
    print(end = "it\n")
main()